File: semestre-1/algoritmos/lista-1/python/MR-Alg-01-Ex-06.py
suco = float(input("Digite o valor do suco: "))
prato_principal = float(input("Digite o valor do prato principal: "))
sobremesa = float(input("Digite o valor da sobremesa: "))

total = (suco + prato_principal + sobremesa) * 1.1

print(f"O valor total da refeição, junto com a taxa de serviço é de R$ {total:.2f}")

File: semestre-1/algoritmos/lista-1/c/MR-Alg-01-Ex-06.c
#include <stdio.h>

void main() {
   float suco, prato_principal, sobremesa;

   printf("Digite o valor do suco: ");
   scanf("%f", &suco);

   printf("Digite o valor do prato principal: ");
   scanf("%f", &prato_principal);

   printf("Digite o valor da sobremesa: ");
   scanf("%f", &sobremesa);

   float total = (suco + prato_principal + sobremesa) * 1.1;
   printf("O valor total da refeição, junto com a taxa de serviço é de R$ %.2f\n", total);
}